### ISSUE TITLE
Meeting schedule

### ISSUE THREAD

**Issue by OpesanyaAdebayo** [nodejsnigeria/admin#1]
For productivity's sake, @nodejsnigeria/admin needs to meet periodically to review ongoing operations and also plan ahead for future ones. My suggestion:

- Frequency: Bi-weekly
- Medium: Zoom
- Length: 1hour
- Day of the week: Sunday
- Time: 8pm

Please let's conclude on this as soon as possible. THanks

**Comment by segunadeleye** [comment]
Don't you think we should meet weekly for now being that it's still the starting phase and we need to put a lot of things in place as quickly as possible?

**Comment by OpesanyaAdebayo** [comment]
You're correct. I'm definitely +1 on meeting weekly. Thank you.

**Comment by OpesanyaAdebayo** [comment]
Where are we on this? It's been almost 2 weeks. Are we okay with:
- Frequency: Weekly
- Medium: Zoom
- Length: 1hour
- Day of the week: Sunday
- Time: 8pm
@nodejsnigeria/admin

**Comment by segunadeleye** [comment]
Perfect for me.

**Comment by OpesanyaAdebayo** [comment]
Okay. Great. In that case, let us have our meeting this Saturday. Not sure if @pmbanugo will be able to make it, but let's start off with something.

I'll create an agenda from the issues across the repos and share with @nodejsnigeria/admin.

**Comment by segunadeleye** [comment]
Awesome. By the way, would our meeting agenda been created as issues.

Also, we should take down minutes (or notes) of our meetings. We can easily refer to whatever we discuss in the meetings.

**Comment by OpesanyaAdebayo** [comment]
Yes. The agenda will be created as issues.

+1 on the minutes of the meeting. We'll create a folder in this repo and upload each meeting minute/note as a separate markdown doc.

**Comment by pmbanugo** [comment]
this saturday is fine. Can we shorten to 30 mins? I don't see the need for weekly. I'd say once a month while actions/plans are communicated through issues here or slack. Though we need to deliberate some things, maybe we start with 2x a month. ?

**Comment by OpesanyaAdebayo** [comment]
If we clear all the items in the meeting agenda before 1hr (which we indeed should), then there's no point extending the meeting till we exhaust the time completely. For me, 1hr is an upper limit and it's fine if we don't exhaust it.

Well, I still advocate for weekly for the following reasons:
- Considering that we are in our beginning stages, a lot of collaborative thought has to be put into this to get it to a place where we have created self-functioning systems from our ideas.
- Things are still moving quite slowly. For example, it has taken almost 2 weeks just to finalize this. If things improve on this note and the issue of frequency is reopened, then we can definitely reduce it if we need to.
- The meeting is where we get the chance to zoom out of the details and see things on a wholistic level, which is important for us as the leaders of this. On some weeks, it's the only chance we have because of how much personal work we have to do. I don't think that dedicating one hour a week to it is too much.

Regardless, I say we should still run this for two consecutive weeks and then revisit this. Are we all game on this? @segunadeleye @pmbanugo
